src/blockchain/download-augur-logs.ts

```diff
@@ -7,7 +7,7 @@ import { logProcessors } from "./log-processors";
 
 export function downloadAugurLogs(db: Knex, augur: Augur, fromBlock: number, toBlock: number, callback: ErrorCallback): void {
   console.log("Getting Augur logs from block " + fromBlock + " to block " + toBlock);
-  augur.events.getAllAugurLogs({ fromBlock, toBlock }, (err: string|Object, allAugurLogs?: AugurLogs) => {
+  augur.events.getAllAugurLogs({ fromBlock, toBlock }, (err: string|object, allAugurLogs?: AugurLogs) => {
```

Comment:
I believe `err: string|object` should be `err: string|object|null` here, since `null` is (apparently) a JS `Object` but *not* a TS `object` :confused: